app/Rules/ValidOneChoiceOptions.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Rules;
+
+use Illuminate\Contracts\Validation\Rule;
+
+class ValidOneChoiceOptions implements Rule
+{
+    /**
+     * Create a new rule instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        //
+    }
+
+    /**
+     * Determine if the validation rule passes.
+     *
+     * @param  string  $attribute
+     * @param  mixed  $value
+     * @return bool
+     */
+    public function passes($attribute, $value)
+    {
+        $rightAnswers = 0;
+
+        if (is_array($value) && count($value) >= 2) {
+            foreach ($value as $option) {
+                if (@$option['is_right']) {
+                    $rightAnswers++;
+                }
+            }
+
+            if ($rightAnswers == 1) {
+                return true;
+            }
```

Comment:
```suggestion
            return $rightAnswers === 1;
```

Reply:
This is just a suggestion, it's not required to change. I think this way is shorter and more beautiful.